image_provisioner/playbooks/roles/aos-ansible/tasks/main.yaml

```diff
@@ -1,9 +1,20 @@
 ---
-- name: customize aos-ansible inventory file
-  template: src=hosts.j2 dest={{ aos_ansible_inventory }}
+- block:
+   - name: customize aos-ansible inventory file
+     template: src=hosts.j2 dest={{ aos_ansible_inventory }}
+ 
+   - name: run aos-ansible
+     shell: ansible-playbook -i {{ aos_ansible_inventory }} {{ aos_ansible_path }}/playbooks/aws_install_prep.yml --connection=local
```

Comment:
Is there another way to toggle between having a local connection other than this command line parameter, that way we reduce the duplication here?